frontend/src/components/templates/Form/index.js

```diff
@@ -40,6 +41,7 @@ export const Form = ({
   const [hasNotFoundError, setHasNotFoundError] = useState(false);
   const navigate = useNavigate();
   const { goBackToList } = useListItemNavigation();
+  const refAlert = useRef(null);
```

Comment:
I'd rather use `alertRef` for better english

Reply:
Sure ! I've changed it 👍🏻